src/main/kotlin/io/snyk/plugin/ui/SnykSettingsDialog.kt

```diff
@@ -202,7 +205,7 @@ class SnykSettingsDialog(
     }
 
     private fun initializeUiComponents() {
-        rootPanel.layout = UIGridLayoutManager(10, 1, JBUI.emptyInsets(), -1, -1)
+        rootPanel.layout = UIGridLayoutManager(11, 1, JBUI.emptyInsets(), -1, -1)
```

Comment:
Why do you need to add a row to the root panel? Is this really necessary?

Reply:
Fixed